New package: DyMat v1.0.1

JuliaRegistrator opened the pull request:
- Registering package: DyMat
- Repository: https://github.com/wlsie/DyMat.jl
- Created by: @wlsie
- Version: v1.0.1
- Commit: b7e224b7a87c5385b8dd8e5e136537b9733cf8c9
- Reviewed by: @wlsie
- Reference: https://github.com/wlsie/DyMat.jl/issues/2#issuecomment-2498773524
- Description: Read Modelica result files with Julia
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835986415faaca43636f0062b5f1f4849b93c6e22e31916baf6c7231926ee9edd237168fdaa8d191c62dd1682512b8f6c76c177c297d79894b1b6e5883649d5255f7bcee91e67e8b83d257609a36da3b27d61ad9bf8cb40f9d85221e843914a6b45cf3e7aabbab7dfd58ac1732867132ff772 -->

wlsie commented:
Please can you help me to merge? I updated my Project.toml file by adding the version for the package MAT in compat section and merged it in the main repo. Then I added comment in the issue to retrigger the bot. However, it says that the version number and commit are not matching. In the instructions, it is mentioned that I do not update the version number.

GunnarFarneback commented:
[noblock]

You need to make the new registration from a commit which includes the Project.toml update.

JuliaRegistrator committed:
New package: DyMat v1.0.1

UUID: 6c8d767c-9b47-4194-a872-a6be4481ec04
Repo: https://github.com/wlsie/DyMat.jl.git
Tree: 13af8cc1c3b0776108f2956d11b3406fddb302d3

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
